### PR TITLE
feat: add option to set macOS client as default for torrent files

### PR DESCRIPTION
**Pull request by nevack**
I think this was missing out.
I confess, I use other torrent clients from time to time, for no particular reason.
And some of them like to set itself as default app without even bothering to ask the user.

XIB diff as always is botched, so take a look at IB preview:

image

### CODE REVIEW

File: macosx/PrefsController.mm [hunk 1]
```diff
@@ -836,14 +840,57 @@ - (void)resetWarnings:(id)sender
     //[fDefaults removeObjectForKey: @"WarningLegal"];
 }
 
-- (void)setDefaultForMagnets:(id)sender
+- (IBAction)setDefaultForMagnets:(id)sender
```
Comment (nevack) [inline macosx/PrefsController.mm:843]: /note to self/
This way you can make action to be discoverable by Interface Builder and jump directly from code to the view that calls this action.

**Review by Coeur — APPROVED** [review]
Good for 4.1.0.

**Comment by nevack** [comment]
@ckerr I'm pinging you on this one, as I have another PR that is based on the changes here.
I think GitHub has no concept of parent/dependent PRs, unfortunately.

**Comment by nevack** [comment]
@ckerr ping again :smile:, good to go!

**Pull request merged**